README.md

```diff
@@ -20,70 +23,194 @@ In addition, "drop-ins" will be loaded from `.conf` files in
 `/usr/lib/systemd/zram-generator.conf.d/`, etc.
 
 The main configuration file is read before any of the drop-ins and has
-the lowest precedence; entries in the drop-in files override entries
-in the main configuration file.
+the lowest precedence; entries in the drop-in files override entries in
+the main configuration file.
 
 See systemd.unit(5) for a detailed description of this logic.
 
 See `zram-generator.conf.example` for a list of available settings.
 
-### Swap devices
+## Swap devices
 
 Create `/etc/systemd/zram-generator.conf`:
 
-```ini
+``` ini
 # /etc/systemd/zram-generator.conf
 [zram0]
 zram-fraction = 0.5
 ```
 
-A zram device will be created for each section. No actual
-configuration is necessary (the default of `zram-fraction=0.5` will be
-used unless overriden), but the configuration file with at least one
-section must exist.
+A zram device will be created for each section. No actual configuration
+is necessary (the default of `zram-fraction=0.5` will be used unless
+overriden), but the configuration file with at least one section must
+exist.
 
-### Mount points
+## Mount points
 
-```ini
+``` ini
 # /etc/systemd/zram-generator.conf
 [zram1]
 mount-point = /var/tmp
 ```
 
-This will set up a /dev/zram1 with ext2 and generate a mount unit for /var/tmp.
+This will set up a /dev/zram1 with ext2 and generate a mount unit for
+/var/tmp.
 
-### Rust
+## Rust
 
 The second purpose of this program is to serve as an example of a
 systemd generator in rust. Details are still being figured out.
 
-### Installation
+## Installation
+
+### Package Manager
 
 It is recommended to use an existing package:
 
-* Fedora: `sudo dnf install zram-generator-defaults` (or `sudo dnf install zram-generator` to install without the default configuration)
-* Debian: packages provided by nabijaczleweli, see https://debian.nabijaczleweli.xyz/README.
-* Arch: `sudo pacman -S zram-generator` (or https://aur.archlinux.org/packages/zram-generator-git/ for the latest git commit)
+#### Fedora 33+
+
+``` bash
+sudo dnf config-manager --add-repo https://download.opensuse.org/repositories/home:alvistack/Fedora_33/home:alvistack.repo
+sudo dnf install zram-generator zram-generator-defaults
```

Comment:
There is no point in recommending personal packages when there's an official build in the distribution. We did that for Debian only because there was no official package. As soon as the official package is provided, the readme would be updated to point to that.